pkgs/development/libraries/librest/default.nix

```diff
@@ -28,12 +28,18 @@ stdenv.mkDerivation rec {
     sha256 = "0513aad38e5d3cedd4ae3c551634e3be1b9baaa79775e53b2dba9456f15b01c9";
   };
 
+  # pkg-config doesn't look in $PATH if strictDeps is on
```

Comment:
We prefer having all the phases in the bottom, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Also can you please move the comment inside the string so that it does not get uncoupled if more commands are added?